src/browser/BrowserPasskeysConfirmationDialog.cpp

```diff
@@ -53,21 +52,22 @@ void BrowserPasskeysConfirmationDialog::registerCredential(const QString& userna
                                                            const QList<Entry*>& existingEntries,
                                                            int timeout)
 {
-    m_ui->firstLabel->setText(tr("Do you want to register Passkey for:"));
+    m_ui->firstLabel->setText(tr("Do you want to register passkey for:"));
     m_ui->relyingPartyLabel->setText(tr("Relying Party: %1").arg(relyingParty));
     m_ui->usernameLabel->setText(tr("Username: %1").arg(username));
+    m_ui->updateButton->setVisible(true);
     m_ui->secondLabel->setText("");
 
     if (!existingEntries.isEmpty()) {
```

Comment:
instead of isEmpty I think it is better to check the existence of the Entry username
if the list existingEntries contains the username "Update" else "Add to existing entry"

Reply:
So iterate the list of existing entries and search for a username match? I can get behind that approach.